### DISCUSSION TITLE
Settings -> Connect with Proton: What does it do?

### DISCUSSION THREAD

**Discussion by Redsandro** [simple-login/app#1185] [Q&A] [answered]
I may have missed a newsletter or blog, but what is this for? What exactly happens if you have a Proton account? What is the benefit? I haven't tried it out myself because I don't know what it does.

:bulb: It would be helpful if the setting would have a small line explaining what it does.

**Comment by nguyenkims** [comment]
@Redsandro we'll make sure to add a small section to explain what are its benefit in the next version.

**Reply by nguyenkims** [reply]
The change has been deployed.

**Reply by Redsandro** [reply]
I don't see it yet, but I thank you in advance on behalf of all future users! :+1:

**Comment by JasonMoriarty** [comment, marked as answer]
It already been added? I guess it will check if you have Proton Unlimited, Business, and Visionary plans. If you have them, you can get SL premium for free.

**Reply by Redsandro** [reply]
Really! Wow that's nice. I wish I saw this feature just a little bit earlier because I paid for both. :sweat_smile: 
It's not the worst thing. SL deserves it. Great service.

**Reply by JasonMoriarty** [reply]
I tried it 5 min ago. It works. The setting said "my SL premium is managed by proton"

**Reply by Redsandro** [reply]
Mine doesn't say anything but I guess it will when my SL premium subscription runs out in 11 months.

**Reply by JasonMoriarty** [reply]
On the top of the setting.
![image](https://user-images.githubusercontent.com/105703258/180903240-a00f4cae-55a9-47c2-8540-d7ce48fc64b9.png)

**Reply by Redsandro** [reply]
It is [no longer free for _plus_, only for _unlimited_ or _business_](https://github.com/simple-login/app/commit/4f4a098b9b8f8af267a11cfc1ae1f1acfed03677).

(While I'm satisfied with this knowledge, you may want to edit your original answer for future readers because it shows up at the top as accepted answer. :slightly_smiling_face:)

**Reply by JasonMoriarty** [reply]
yea changed it

**Reply by Redsandro** [reply]
This will make it more attractive to upgrade to Proton Unlimited if you have Plus and SL.

**Reply by JasonMoriarty** [reply]
I personally think it won't. For those who don't really need unlimited plan. They can just go get a SL PREMIUM.  It's only $30 per year.  But Proton unlimited needs $10 per month.  It much more expensive.

**Reply by Redsandro** [reply]
I guess it depends. It would for me. Proton unlimited costs $10 per month, but Proton Plus and SL Premium cost combined $7.50 per month. So for $2.50 per month extra you can get the same, but with additional fast VPN and 500 GB storage. Had I known this, I would probably upgrade to Premium. Now I have SL separate, a VPN provider separate and cloud storage separate.

**Reply by JasonMoriarty** [reply]
Make sense.
